pubsub/natspubsub/nats.go

```diff
@@ -160,24 +159,30 @@ func (*topic) ErrorCode(err error) gcerrors.ErrorCode {
 }
 
 type subscription struct {
-	nc   *nats.Conn
-	nsub *nats.Subscription
-	err  error
+	nc      *nats.Conn
+	nsub    *nats.Subscription
+	ackFunc func()
+	err     error
 }
 
 // CreateSubscription returns a *pubsub.Subscription representing a NATS subscription.
 // TODO(dlc) - Options for queue groups?
-func CreateSubscription(nc *nats.Conn, subscriptionName string) *pubsub.Subscription {
-	return pubsub.NewSubscription(createSubscription(nc, subscriptionName), nil)
+func CreateSubscription(nc *nats.Conn, subscriptionName string, ackFunc func()) *pubsub.Subscription {
```

Comment:
You should move or copy your explanation of ackFunc here. It's more important that it be here.

Reply:
Good point, I meant to do that but forgot. Done.